src/main/java/Duke/Task/Deadline.java

```diff
@@ -0,0 +1,38 @@
+package Duke.Task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represent a deadline object with a deadline timing.
+ */
+public class Deadline extends Task {
+    
+    protected LocalDateTime by;
+
+    /**
+     * Construct a deadline object.
+     * @param description title of the deadline.
```

Comment:
In Javadoc comments, there should be an empty line between description and parameter section and the first word of the params' description should be capitalised.